<launch>

  <!-- change this next line to include the launch file for your robot's hardware drivers: -->

 <arg name="simulation" default="$(optenv TURTLEBOT_SIMULATION false)"/>
 <param name="/use_sim_time" value="$(arg simulation)"/>

  <arg name="urdf_file" default="$(find xacro)/xacro.py '$(find explodroid_description)/urdf/explodroid_base.xacro'" /> 
<!--  <arg name="urdf_file" default="$(find xacro)/xacro.py '$(find arlobot_description)/urdf/common.urdf.xacro'" /> -->


  <param name="robot_description" command="$(arg urdf_file)" />

  <!-- important generally, but specifically utilised by the current app manager -->
  <param name="robot/name" value="$(optenv ROBOT turtlebot)"/>
  <param name="robot/type" value="turtlebot"/>


  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
    <param name="publish_frequency" type="double" value="5.0" />
  </node>

s
  <rosparam param="base_width">0.4</rosparam>
  <rosparam param="ticks_meter">17420</rosparam>


  <node name="launchpad_node" pkg="explodroid_bringup" type="launchpad_node.py">
    <rosparam file="$(find explodroid_bringup)/param/board_config.yaml" command="load" />
  </node>



  <node pkg="explodroid_bringup" type="twist_to_motors.py" name="twist_to_motors" output="screen"/>
  <node pkg="explodroid_bringup" type="diff_tf.py" name="diff_tf" output="screen"/> 


</launch>
